detekt.yml

```diff
@@ -20,3 +20,13 @@ complexity:
     threshold: 100 # up from 60
   TooManyFunctions:
     active: false
+  ReturnCount:
+    active: false
+  MaxLineLength:
```

Comment:
ugh i want to keep this, but i think we need to change from 4 space indent to 2 first.